01 Key Concepts/01 Getting Started/01 Welcome.html

```diff
@@ -1,4 +1,7 @@
 <meta name="tag" content="using quantconnect">
-<p><b>Welcome to QuantConnect.</b>
+<p class="english"><b>Welcome to QuantConnect.</b>
 	 We are an open source, community driven algorithmic trading platform. Our trading engine is powered by <a href="https://www.quantconnect.com/lean" target="_BLANK">LEAN</a>, a cross platform, multi-asset technology which brings cutting edge finance to the open source community. We support <b>C#</b>, <b>Python</b> and <b>F#</b> programming languages making us a truly open platform.
-</p>
\ No newline at end of file
+</p>
+<p class=“mandarin”><b>欢迎来到QuantConnect。</b>
```

Comment:
Make sure to use simple double quotes `"`

Reply:
Okay. Thanks.